css/src/core/animations.scss

```diff
@@ -24,3 +24,24 @@
 		transform: scale(1);
 	}
 }
+
+@keyframes slide-up-fade-out {
+	100% {
+		transform: translateY(-100%);
+		opacity: 0;
+	}
+}
+
+@keyframes fade-out {
+	100% {
+		opacity: 0;
+	}
+}
+
```

Comment:
Can these be wrapped in our reduced motion no preference? 
```scss
@media (prefers-reduced-motion: no-preference) {
.animation-fade {
	animation: fade-out 0.4s ease-in forwards;
}

.animation-slide-up {
	animation: slide-up-fade-out 0.4s ease-in forwards;
}
}
```
This should prevent these classes from having an effect if a user has set their preference to reduce.

https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion

Reply:
Updated with Ben's suggestion, check the comment below